test/telemetry_influx_db_test.exs

```diff
@@ -423,9 +592,49 @@ defmodule TelemetryInfluxDBTest do
     assert_tags(config, tags)
   end
 
+  defp do_assert_reported(%{version: :v2} = config, name, values, tags) do
+    results =
+      eventually(fn ->
+        res = query(config, name)
+
+        if empty_result?(config, res) do
+          false
+        else
+          res
+        end
+      end)
+
+    tag_values = Map.values(tags)
+    tag_keys = Map.keys(tags)
+
+    assert Enum.all?(results, fn result ->
+             measurement = Map.get(result, "_measurement")
+
+             has_tag_keys = Enum.all?(tag_keys, fn tag_key -> Map.has_key?(result, tag_key) end)
+
+             result_tag_values =
+               tags
+               |> Map.keys()
+               |> Enum.map(fn key -> Map.get(result, key) end)
+
+             measurement == name and has_tag_keys and result_tag_values == tag_values
```

Comment:
it is a NIT but such representation:
`(measurement == name) and has_tag_keys and (result_tag_values == tag_values)`
seems a little bit more readable for me

Reply:
I agree with you on this one, but the Elixir formatter does not; it removes the parens when I try to add them.